pilot/pkg/model/telemetry_logging.go

```diff
@@ -172,13 +174,23 @@ func telemetryAccessLog(push *PushContext, fp *meshconfig.MeshConfig_ExtensionPr
 	return al
 }
 
-func tcpGrpcAccessLogFromTelemetry(push *PushContext, prov *meshconfig.MeshConfig_ExtensionProvider_EnvoyTcpGrpcV3LogProvider) *accesslog.AccessLog {
+func filterStateObjectsToLog(proxy *Proxy) []string {
+	if proxy.VersionGreaterAndEqual(&IstioVersion{Major: 1, Minor: 24, Patch: 0}) {
```

Comment:
```suggestion
	if proxy.VersionGreaterAndEqual(&IstioVersion{Major: 1, Minor: 24, Patch: -1}) {
```

Reply:
GreaterAndEqual, so I think it's fine to use `1.24.0`?